src/store/RelayStore.js

```diff
@@ -13,160 +13,6 @@
- *      // Failure occurred.
- *    } else if (readyState.ready) {
- *      // Queries are at least partially resolvable.
- *      if (readyState.done) {
- *        // Queries are completely resolvable.
- *      }
- *    }
- *  }
- *
- */
-var RelayStore = {
-
-  /**
-   * Primes the store by sending requests for any missing data that would be
-   * required to satisfy the supplied set of queries.
-   */
-  primeCache(
-    querySet: RelayQuerySet,
-    callback: ReadyStateChangeCallback
-  ): Abortable {
-    return GraphQLQueryRunner.run(querySet, callback);
-  },
-
-  /**
-   * Forces the supplied set of queries to be fetched and written to the store.
-   * Any data that previously satisfied the queries will be overwritten.
-   */
-  forceFetch(
-    querySet: RelayQuerySet,
-    callback: ReadyStateChangeCallback
-  ): Abortable {
-    return GraphQLQueryRunner.forceFetch(querySet, callback);
-  },
-
-  /**
-   * Reads query data anchored at the supplied data ID.
-   */
-  read(
-    node: RelayQuery.Node,
-    dataID: DataID,
-    options?: StoreReaderOptions
-  ): ?StoreReaderData {
-    return readRelayQueryData(queuedStore, node, dataID, options).data;
-  },
-
-  /**
-   * Reads query data anchored at the supplied data IDs.
-   */
-  readAll(
-    node: RelayQuery.Node,
-    dataIDs: Array<DataID>,
-    options?: StoreReaderOptions
-  ): Array<?StoreReaderData> {
-    return dataIDs.map(
-      dataID => readRelayQueryData(queuedStore, node, dataID, options).data
-    );
-  },
-
-  /**
-   * Reads query data, where each element in the result array corresponds to a
-   * root call argument. If the root call has no arguments, the result array
-   * will contain exactly one element.
-   */
-  readQuery(
-    root: RelayQuery.Root,
-    options?: StoreReaderOptions
-  ): Array<?StoreReaderData> {
-    const storageKey = root.getStorageKey();
-    var results = [];
-    forEachRootCallArg(root, identifyingArgValue => {
-      var data;
-      var dataID = queuedStore.getDataID(storageKey, identifyingArgValue);
-      if (dataID != null) {
-        data = RelayStore.read(root, dataID, options);
-      }
-      results.push(data);
-    });
-    return results;
-  },
-
-  /**
-   * Reads and subscribes to query data anchored at the supplied data ID. The
-   * returned observable emits updates as the data changes over time.
-   */
-  observe(
-    fragment: RelayQuery.Fragment,
-    dataID: DataID
-  ): Observable<?StoreReaderData> {
-    var fragmentPointer = new GraphQLFragmentPointer(
-      fragment.isPlural()? [dataID] : dataID,
-      fragment
-    );
-    return new RelayQueryResultObservable(queuedStore, fragmentPointer);
-  },
-
-  update(
-    mutation: RelayMutation,
-    callbacks?: RelayMutationTransactionCommitCallbacks
-  ): void {
-    var transaction = new RelayMutationTransaction(mutation);
-    transaction.commit(callbacks);
-  }
-};
-
-module.exports = RelayStore;
+module.exports = new RelayContext();
```

Comment:
We should probably hook the methods called on this instance to show a deprecation warning.